app/src/main/res/values/strings.xml

```diff
@@ -1066,6 +1066,8 @@
     <string name="preferences_delete_browsing_data_cached_files_subtitle">Frees up storage space</string>
     <!-- Title for the site permissions item in Delete browsing data -->
     <string name="preferences_delete_browsing_data_site_permissions">Site permissions</string>
+    <!-- Title for the downloads item in Delete browsing data -->
```

Comment:
Could we change this description a bit? Proposal: Title for the deleting downloads item in Delete browsing data 

Update: oh strings here are all following that pattern.. we should try to change those at some point to make them a bit more descriptive 😄